data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+				accept
+			label decline
+			`	"I see," he says. He gives you the coordinates he needs to go to and you drop him off before returning to the hangars.`
+			`	Kirrie and Iplora come back hours later and contact Tugroob, who says you can pick him up where you left him. Once he's back in your ship, he tells you he found nothing worthwhile.`
+
+mission "HI2-FS"
+	name "Investigate <planet>"
+	description "Transport the Heliarch trio to <destination> so that they can investigate signs of criminal activity."
+	to offer
+		has "Heliarch Investigation 1: done"
+	source "Ring of Friendship"
+	destination "Fourth Shadow"
+	to fail
+		or
+			has "Heliarch Investigation 2: declined"
+			has "Heliarch Investigation 2: aborted"
+			has "Heliarch Investigation 2 :failed"
+	on complete
+		conversation
+			`For this planet, the trio informs you that they cannot spend too much time in one place considering how densely populated it is. Instead, they hand you the coordinates of dozens of factories, with the instructions to take them to each factory in whichever order you'd prefer.`
+			`	"Surprise inspections, we shall perform," Kirrie says as the three prepare their equipment and weapons. You spend hours flying around the planet dropping one agent off in a factory, running off to another, then another, then picking up the first of the three, and continuing the process.`
+			`	After picking up Tugroob from the last factory, they all analyze their findings one last time. They do this by going over each individual factory, mechanically running down a long list of "possible abnormalities" for each, leaving a mark on the side, and briefly writing a description if any such "abnormality" was noted in a factory.`
+			`	"Unauthorized power surges to some replicators, we found. Missing as well, some records of storage transport were," Iplora says. You land on a Heliarch base, and they hand all their findings to the agents there before returning to your ship. As they head to their bunks, you see several Heliarch ships fly off in the direction of some of the factories.`
+
+mission "HI2-IW"
+	name "Investigate <planet>"
+	description "Transport the Heliarch trio to <destination> so that they can investigate signs of criminal activity."
+	to offer
+		has "Heliarch Investigation 1: done"
+	source "Ring of Friendship"
+	destination "Into White"
+	to fail
+		or
+			has "Heliarch Investigation 2: declined"
+			has "Heliarch Investigation 2: aborted"
+			has "Heliarch Investigation 2 :failed"
+	on complete
+		conversation
+			`Kirrie, already donning his winter coat, instructs you to land near the equator. He says him and Tugroob will conduct some subtle interrogations, stating that it's 'all they can do' in this climate.`
```

Comment:
The physical climate is a limiting factor? Seems odd.

Reply:
I mean, the planet is very, very cold, and the Kimek and Arachi are known to not do well in such climate.
As for the clothing in spite of interrogations likely occurring indoors... well whenever my family and I traveled to a cold place, usually we wore the winter clothing even if we were indoors. Unless we had just gotten back to the hotel and knew we were only going to leave again in hours, we'd keep the clothes on.
I just don't think it's that unusual to wear winter clothes when on a planet as cold as Into White?